lib/Service/UserMigrationService.php

```diff
@@ -254,6 +275,56 @@ protected function importAppsSettings(IUser $user,
 		}
 	}
 
+	/**
+	 * @throws UserMigrationException
+	 */
+	public function queueExport(IUser $user, array $migrators): void {
```

Comment:
I’m a bit uneasy about putting job handilng logic in UserMigrationService, since the jobs are themselves calling export methods of UserMigrationService. Previous state was that UserMigrationService was for doing the actual export/import work, and background jobs allowed to encapsulate that in a job.
At the very least rename methods to queueExportJob and queueImportJob.

Reply:
Renamed to `queueExportJob` and `queueImportJob`

If a refactor is needed we may do that outside of this PR :)